src/Tickets/Commerce/Module.php

```diff
@@ -214,6 +214,37 @@ public function __construct() {
 	 */
 	protected $is_loaded = false;
 
+	/**
+	 * Attendees by ticket id.
+	 *
+	 * @since TBD
+	 *
+	 * @var array $attendees_by_ticket_id List of attendees.
+	 */
+	protected $attendees_by_ticket_id = [];
+
+	/**
+	 * Attendees by order id.
+	 *
+	 * @since TBD
+	 *
+	 * @param int          $ticket_id Ticket ID.
+	 * @param WP_Post|null $attendee  Attendee object.
+	 */
+	public function add_attendee_by_ticket_id( $ticket_id, $attendee = null ) {
+		// Go ahead and set an empty array if it doesn't exist.
+		if ( ! isset( $this->attendees_by_ticket_id[ $ticket_id ] ) ) {
+			$this->attendees_by_ticket_id[ $ticket_id ] = [];
+		}
+
+		// If no attendee, bail.
+		if ( ! $attendee ) {
+			return;
+		}
+
+		$this->attendees_by_ticket_id[ $ticket_id ][] = $attendee;
+	}
+
```

Comment:
Do not add new methods to the Module. This needs to live in the Commerce Attendee class.

Reply:
I actually added this to the main `Tribe__Tickets__Tickets` class so it could be used within any of the providers.